ChemicalFeatureAnalysis/analysis/ChemTreeMap/examples.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python
+#
+# Copyright (C) 2016 Jing Lu <ajingnk@gmail.com>
```

Comment:
@ijmiller2 how many of these files are from the ChemTreeMap code base?  If one or more files are from a different original source, I recommend adding a license notice to the main readme of this repo that links back to the original source and states the license and author.  See example language here: https://github.com/gitter-lab/SINGE#licenses

Reply:
Hey @agitter,

The only file here from the ChemTreeMap code base is this one (`examples.py`), which I had to modify a fair amount to work for our purposes (instructions on his man page were not very complete). The other two .py files are just a bit of pre-processing code to generate and filter the input data required for the ChemTreeMap visualization. The .txt files are input and pre-input data.

I can update the pull request to put this in the main ReadMe:

The file `examples.py` has been modified from the original version provided by the [ChemTreeMap docker container](http://ajing.github.io/ChemTreeMap/). The original third-party code is available under the Apache License, Copyright Copyright © 2016 Jing Lu <ajingnk@gmail.com>

Does that look good?

Reply:
Thanks @ijmiller2, that text looks perfect.  If you don't want it in the main readme, it would be reasonable to have it in a new `ChemicalFeatureAnalysis/analysis/README.md` if you don't want it on the front page.

Reply:
Just noticed the double `Copyright Copyright`.  Everything else is good.